### PR TITLE
feat(docs): integration docs

### LINKED ISSUE #299: Write Integrations docs

### PR DESCRIPTION
**Pull request by VojtechVidra**
closes #299

### CODE REVIEW

**Review by OPesicka — APPROVED** [review]

File: apps/docs/src/pages/analytics/integrations.mdx [hunk 1]
```diff
@@ -1,30 +1,30 @@
 ---
 title: Integrations
-description: Integrate with your favorite tools to get the most out of your analytics.
+description: Integrate with any analytics tool to get the full picture of your users' behavior.
 ---
 
 # Integrations
 
-Flows automatically track how users interact with your flows. You can either view this data in aggregated form in the dashboard [insights section](/analytics/flow-insights) or integrate with your favorite analytics tools to get more control and flexibility.
+Flows automatically track how users interact with your flows to help you better understand how users are engaging with your product. When the built-in [insights](/analytics/flow-insights) are not enough, you can easily integrate with almost any analytics tool to get the full picture of your users' behavior.
```
Comment (VojtechVidra) [inline apps/docs/src/pages/analytics/integrations.mdx:8]: `When the built-in` should probably be `If the built-in`

File: apps/docs/src/pages/analytics/integrations.mdx [hunk 2]
```diff
@@ -69,8 +69,32 @@ interface TrackingEvent {
...
+
 - `startFlow`: When a user starts a flow.
 - `nextStep`: When a user goes to the next step in a flow.
 - `prevStep`: When a user goes to the previous step in a flow.
 - `finishFlow`: When a user finishes a flow.
-- `cancelFlow`: When a user exits the flow without completion.
\ No newline at end of file
+- `cancelFlow`: When a user exits the flow without finishing it.
+
+## Examples
```
Comment (VojtechVidra) [inline apps/docs/src/pages/analytics/integrations.mdx:80]: Maybe we can mention Mixpanel directly in heading?

File: apps/docs/src/pages/analytics/integrations.mdx [hunk 3]
```diff
@@ -69,8 +69,32 @@ interface TrackingEvent {
...
+
+Here is an example of how you can send events from Flows to Mixpanel for analysis:
+
+```js
+init({
```
Comment (VojtechVidra) [inline apps/docs/src/pages/analytics/integrations.mdx:85]: Add `import mixpanel from 'mixpanel-browser'`

File: apps/docs/src/pages/analytics/integrations.mdx [hunk 4]
```diff
@@ -69,8 +69,32 @@ interface TrackingEvent {
 
 ## Events
```
Comment (VojtechVidra) [inline apps/docs/src/pages/analytics/integrations.mdx:70]: I think this section could be directly after setup

**Pull request merged**